collector/test/ConfigLoaderTest.cpp

```diff
@@ -42,34 +42,16 @@ TEST(CollectorConfigTest, TestYamlConfigToConfigMultiple) {
   }
```

Comment:
Ideally this would have the same behavior as before and no changes to the unit tests would have to be made. However, there the old code had checks that the networking field was present and not blank and with the new more general approach this check is not done. Also the old allowed for the yaml to have invalid values, but `JsonStringToMessage` does not allow that.